packages/pglite/src/live/index.ts

```diff
@@ -253,6 +252,14 @@ const setup = async (pg: PGliteInterface, _emscriptenOpts: any) => {
               changes = await tx.query<any>(
                 `EXECUTE live_query_${id}_diff${stateSwitch};`,
               )
+
+              // Switch state
+              stateSwitch = stateSwitch === 1 ? 2 : 1
```

Comment:
super nit: the switching of state could generally be `stateSwitch + 1 % 2`  for `0` and `1` (and +1 for the current one)